workflow.rst

```diff
@@ -265,6 +265,36 @@ registry in the constructor::
         }
     }
 
+Or use the typed + named parameter injection::
+
+    use App\Entity\BlogPost;
+    use Symfony\Component\Workflow\WorkflowInterface;
+
+    class MyClass
+    {
+        private $blogPostWorkflow;
+
+        public function __construct(WorkflowInterface $blogPostWorkflow)
```

Comment:
This looks wrong to me, it should be $blogPublishingWorkflow according to the definition https://github.com/symfony/symfony-docs/blob/018b64c26fa07591c551da0619058918899ddc61/workflow.rst#creating-a-workflow

Reply:
:heavy_check_mark: